cmake/macos_installer_deps.cmake

```diff
@@ -1,8 +1,9 @@
 # macos specific cpack stuff goes here
 
+return()
```

Comment:
This looks temporary: I think we should just delete this file (and follow it back to remove includes).